### ISSUE TITLE
Lil' Peepz Stock Characters

### ISSUE THREAD

**Issue by rryan22** [GoAnimate-Wrapper/GoAnimate-Wrapper#66]
Is there Lil' Peepz Stock Characters included with all of these including Mr. Grinch, Santa Claus, Christmas Elf, Ghost Kid, Turkey, DJ, Osama Bin Laden, Red DJ, White DJ, Black DJ, and others!

**Comment by Windows81** [comment]
I don't believe so.  Do you have any of the theme files for these characters?

**Comment by rryan22** [comment]
Yes in action.xml file.

**Comment by rryan22** [comment]
Does all Lil' Peepz Stock Characters coming?
